src/toolbox.js

```diff
@@ -130,19 +130,28 @@ var Toolbox = GObject.registerClass({
         this.setBusy(false);
     }
 
-    addTopic(id, title, iconName, widget) {
-        const topic = new TopicButton({id, title, iconName});
+    addTopic(id, title, iconName, widget, lockscreen = false) {
+        const topic = new TopicButton({id, title, iconName, lockscreen});
         this._topicsList.add(topic);
+
         if (GObject.Object.find_property.call(widget.constructor, 'needs-attention')) {
             widget.bind_property('needs-attention',
                 topic, 'needs-attention', GObject.BindingFlags.SYNC_CREATE);
         }
 
-
         widget.show_all();  // show_all() only propagates to current stack page
         this._topicsStack.add_titled(widget, id, title);
     }
 
+    addTopicKeys(id, appName, lockNum) {
+        const [topicRow, topic] = this._findTopic(id); // eslint-disable-line
```

Comment:
In general, prefer to disable only specific warnings with eslint, e.g. `// eslint-disable-line no-restricted-syntax`. That said, if I remove this comment it looks like the warning is that `topicRow` is unused; you can fix that here with `const [, topic] = this._findTopic(id);` and below with `const [topicRow] = this._findTopic(id);` (you don't have to name all the parameters that you are unpacking from an array, unlike Python)

Reply:
I did try removing the unused parameters (that is how I coded it originally) but it didn't work. I'll try again.